UT4MasterServer/Controllers/SessionController.cs

```diff
@@ -268,41 +263,41 @@ public async Task<IActionResult> KillSessions([FromQuery] string killType)
 		switch (killType.ToUpper())
 		{
 			case "ALL":
-			{
-				// TODO: Check permission and return ErrorResponse: "Sorry your login does not posses the permissions 'account:token:allSessionsForClient DELETE' needed to perform the requested operation"
-				await sessionService.RemoveSessionsWithFilterAsync(EpicID.Empty, user.Session.AccountID, EpicID.Empty);
-				break;
-			}
+				{
+					// TODO: Check permission and return ErrorResponse: "Sorry your login does not posses the permissions 'account:token:allSessionsForClient DELETE' needed to perform the requested operation"
+					await sessionService.RemoveSessionsWithFilterAsync(EpicID.Empty, user.Session.AccountID, EpicID.Empty);
+					break;
+				}
```

Comment:
Fix indentation for better git diff

Reply:
Again, I autoformatted and I did review. Github has an ignore whitespace setting.